Tiltfile

```diff
@@ -0,0 +1,40 @@
+# -*- mode: Python -*-
+
+# set defaults
+BASE_IMAGE_MINIMAL = os.getenv('BASE_IMAGE_MINIMAL', "gcr.io/distroless/base")
+BASE_IMAGE_FULL = os.getenv('BASE_IMAGE_FULL', "debian:buster-slim")
+BUILDER_IMAGE = os.getenv('BUILDER_IMAGE', "golang:1.18-buster")
+HOSTMOUNT_PREFIX = os.getenv('HOSTMOUNT_PREFIX', "/host-")
```

Comment:
I don't think we want to make this configurable (would probably be good to remove from the developer documentation, too, if we have it there). Changing this will break things as deployment doesn't change the mounts accordingly.

Reply:
Sure thing, then I will leave only image naming and tagging related vars.